lib/src/non_fungible_token/io.rs

```diff
@@ -1,22 +1,22 @@
 use crate::non_fungible_token::{royalties::*, token::*};
 use gstd::{prelude::*, ActorId};
 
-#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq)]
+#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone, Copy)]
```

Comment:
Not sure `Copy` is really needed.
```suggestion
#[derive(Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Clone)]
```